code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -186,7 +186,7 @@
 			mob_occupant.Unconscious((mob_occupant.bodytemperature * unconscious_factor) * 2000)
 		if(beaker)
 			if(reagent_transfer == 0) // Magically transfer reagents. Because cryo magic.
-				beaker.reagents.trans_to(occupant, 1, 10 * efficiency) // Transfer reagents, multiplied because cryo magic.
+				beaker.reagents.trans_to(occupant, 1, efficiency * 0.25) // Transfer reagents.

```

Comment:
also wasn't one point of this rebalance to make cryo not multiply reagents? it still will with this, just by a smaller number

Reply:
Multiplying by a number below 1 is the same as dividing. This is the same as dividing by four.

Reply:
a: doesn't efficiency go up to 5, b: you're still multiplying reagents, just by .25 instead of 10